core/domain/suggestion_jobs_one_off.py

```diff
@@ -471,3 +475,74 @@ def reduce(key, values):
             yield (key, len(values))
         else:
             yield (key, values)
+
+
+class DeleteAllQuestionAndInvalidSuggestionsOneOffJob(
```

Comment:
There should be an "Exp" after "Invalid".

Reply:
(I think? Not sure.)

Reply:
This job deletes any general suggestion model which is invalid (i.e, doesn't have corresponding feedback models) I've added more details in the docstring.